assets/docs/snippets/shortname.md

```diff
@@ -0,0 +1 @@
+kgateway
```

Comment:
This conref is used informally in places like resource names, such as a role name in the lambda guide, `kagteway-lambda-role`